IntercomDotnet/Resources/Users.cs

```diff
@@ -47,7 +47,7 @@ public dynamic DeleteWithUserId(string userId)
         [Obsolete("userId is a string in the Intercom API so please use the string-based overload going forward.")]
         public dynamic Get(string email = null, int? userId = null)
         {
-            return Get(email, userId?.ToString());
+            return Get(email, userId.ToString());
```

Comment:
OK, not the end of the world - I prefer the new null coalescing operator when used on objects such as `?.` - of course, the IL generated code from that operator would be closer to writing:

```
return Get(email, userId == null ? null : userId.ToString());
```

But for now I see no reason to change it to that from using Convert, so let's merge anyway!